langchain/src/chains/tests/sql_database_chain.int.test.ts

```diff
@@ -0,0 +1,39 @@
+import { test } from "@jest/globals";
+import { DataSource } from "typeorm";
+import { OpenAI } from "../../llms/openai.js";
+import { SqlDatabaseChain } from "../sql_database/sql_database_chain.js";
+import { SqlDatabase } from "../../sql_database.js";
+
+test("Test SqlDatabaseChain", async () => {
+  const datasource = new DataSource({
+    type: "sqlite",
+    database: ":memory:",
+    synchronize: true,
+  });
+
+  await datasource.initialize();
+  await datasource.query(`
+        CREATE TABLE users (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT, age INTEGER);
+    `);
+  await datasource.query(`
+        INSERT INTO users (name, age) VALUES ('Alice', 20);
+    `);
+  await datasource.query(`
+        INSERT INTO users (name, age) VALUES ('Bob', 21);
+    `);
+  await datasource.query(`
+        INSERT INTO users (name, age) VALUES ('Charlie', 22);
+    `);
```

Comment:
@nfcampos -- FYI couldn't figure out a way to add a bunch of statements to the same `query` call (it only seemed to process the first one). not sure if there's a better way

Reply:
As far as I can tell there is no way to do that in typeorm...